src/main/webapp/admin/AddStaticData.html

```diff
@@ -19,6 +19,28 @@ <h1> Enter in entities one by one to be added to the database here </h1>
   </form>
   <button type = "submit" form = "careerQuizForm" name = "careerQuizSubmit" value = "Submit">Submit</button>
 
+  <p> Promotion Quiz </p>
+  <form action = "/storePromotion" method = "POST" id = "promotionQuizForm" name = "promotionQuizForm">
+    <label>Question:</label><br>
+    <input type = "text" name = "promotionQuestion" required = "true"><br>
```

Comment:
Not a super necessary change, but instead of `required = "true"` you can just put `required` and it all works the same

Reply:
ooo good tip, thanks.